index.bs

```diff
@@ -2212,36 +2216,38 @@ The mechanism for generating scoped credentials, as well as requesting and gener
 platform performs additional processing for each extension that it supports, and augments the [=client data=] as required by the
-extension. In addition, the client collects the authenticator arguments for the above extensions, and passes them to the
+extension. In addition, the client collects the [=authenticator extension input=] values for the above extensions, and passes them to the
 authenticator in the [=authenticatorMakeCredential=] call (for [=registration extensions=]) or [=authenticatorGetAssertion=]
-call (for [=authentication extensions=]). These authenticator arguments are passed as name-value pairs, with the extension
-identifier as the name, and the corresponding authenticator argument as the value. The authenticator, in turn, performs
-additional processing for the extensions that it supports, and augments the [=authenticator data=] as specified by the
-extension.
+call (for [=authentication extensions=]). These authenticator extension arguments are passed as name-value pairs,
+with the [=extension identifier=] as the name, and the corresponding [=authenticator extension input=] as the value.
+The authenticator, in turn, performs additional processing for the extensions that it supports,
+and returns the [=authenticator extension output=] as specified by the extension.
 
 All WebAuthn extensions are optional for both clients and authenticators. Thus, any extensions requested by a [RP] may be
 ignored by the client browser or OS and not passed to the authenticator at all, or they may be ignored by the authenticator.
 Ignoring an extension is never considered a failure in WebAuthn API processing, so when [RPS] include extensions with any API
 calls, they must be prepared to handle cases where some or all of those extensions are ignored.
 
 Clients wishing to support the widest possible range of extensions may choose to pass through any extensions that they do not
-recognize to authenticators, generating the authenticator argument by simply encoding the client argument in CBOR. All
+recognize to authenticators, generating the [=authenticator extension input=] by simply encoding the [=client extension input=] in CBOR. All
 WebAuthn extensions MUST be defined in such a way that this implementation choice does not endanger the user's security or
 privacy. For instance, if an extension requires client processing, it could be defined in a manner that ensures such a naïve
-pass-through will produce a semantically invalid authenticator argument, resulting in the extension being ignored by the
+pass-through will produce a semantically invalid [=authenticator extension input=] value, resulting in the extension being ignored by the
 authenticator. Since all extensions are optional, this will not cause a functional failure in the API operation.
+Likewise, clients can choose to produce a [=client extension output=] value for an extension that it does not understand by encoding the
```

Comment:
> the client extension output is created from the authenticator extension output, not from the result of the client processing steps?

It seems the answer is "no, see 1st parag of [[sctn-client-extension-processing]]"

> Does it mean that we need two client processing algorithms, one to process the input, and another to process the output?

that seems to be addressed by the last parag of [[sctn-client-extension-processing]] in combination with section [[#sctn-authenticator-extension-processing]].

we could perhaps have better linking to the latter sections from this opening [[#extensions]] section.